src/main/kotlin/com/bada/app/config/EmployeeSecurityConfig.kt

```diff
@@ -28,6 +34,8 @@ class ApplicationSecurityConfig(
     override fun configure(auth: AuthenticationManagerBuilder?) {
         auth?.run {
             userDetailsService(employeeDetailsService)
+                .and()
+                .userDetailsService(customerDetailsService)
```

Comment:
Not sure how this is going to work, when two users have the same credentials, please split the configuration into 2 files with `@Order` annotation.

Reply:
Impossible due to engine constraints.

Reply:
Fixed